server/events/command_runner.go

```diff
@@ -224,6 +224,13 @@ func (c *DefaultCommandRunner) RunCommentCommand(baseRepo models.Repo, maybeHead
 		ctx.Log.Err("Error running pre-workflow hooks %s. Proceeding with %s command.", err, cmd.Name.String())
 	}
 
+	if cmd.ForceApply {
+		warningMessage := "⚠️ WARNING ⚠️\n\n You have bypassed all apply requirements for this PR 🚀 . This can have unpredictable consequences 🙏🏽 and should only be used in an emergency 🆘 .\n\n 𝐓𝐡𝐢𝐬 𝐚𝐜𝐭𝐢𝐨𝐧 𝐰𝐢𝐥𝐥 𝐛𝐞 𝐚𝐮𝐝𝐢𝐭𝐞𝐝.\n"
+		if commentErr := c.VCSClient.CreateComment(baseRepo, pullNum, warningMessage, ""); commentErr != nil {
+			log.Err("unable to comment: %s", commentErr)
+		}
+	}
+
```

Comment:
Shouldn't this be wrapped in the feature flag as well? The feature flag call should be idempotent and quick so you could theoretically employ the decorator pattern again here and wrap this.

Reply:
Makes sense. I need to add some unit tests quick but how does this look:  https://github.com/lyft/atlantis/pull/107/commits/bb3607c6e55a6217966a1c7b9bbc367bbf85e363

Reply:
done